src/CMakeLists.txt

```diff
@@ -1,8 +1,11 @@
 add_subdirectory( multio )
 
-if (NOT CMAKE_Fortran_COMPILER_ID STREQUAL "GNU")
-message(STATUS "Building multiom")
-add_subdirectory( multiom )
-else ()
-message(WARNING "Skipping multiom because the GNU Fortran compiler is too slow.")
+if ( HAVE_FORTRAN )
+    if (NOT CMAKE_Fortran_COMPILER_ID STREQUAL "GNU")
+        message(STATUS "Building multiom")
+        add_subdirectory( multiom )
+    else ()
+        message(WARNING "Skipping multiom because the GNU Fortran compiler is too slow.")
+    endif()
+    message(WARNING "Skipping multiom because Fortran is disabled.")
```

Comment:
I think there is an `else ()` missing prior to this line???
Now the message `... Fortran is disabled..` is printed when HAVE_FORTRAN is true

Reply:
Good spot! :^)